packages/arb-token-bridge-ui/src/components/TransferPanel/useTransferReadiness.ts

```diff
@@ -218,19 +218,22 @@ export function useTransferReadiness(): UseTransferReadinessResult {
       gasSummary,
       {
         isSmartContractWallet,
-        isDepositMode
+        isDepositMode: transferMode === 'deposit' || transferMode === 'teleport'
```

Comment:
We have this check in multiple place in this file, it might be better to extract it (if they all are related)

Reply:
i think we want to do inline to make it very explicit so i'll keep it this way